src/test/kotlin/no/nav/arbeidsgiver/altinnrettigheter/proxy/ApplicationTest.kt

```diff
@@ -17,7 +16,7 @@ class ApplicationTest {
     @Autowired
     private val controller: AltinnrettigheterProxyController? = null
 
-    @Test
+    @org.junit.Test
```

Comment:
Hvorfor må vi skrive alt dette? Har vi to konkurrerende testbiblioteker? 🤔

Reply:
Etter oppdatering blir det feil og Intelliji foreslo denne endringen. Skal prøve igjen og vise hva vi får til.